lib/redcord/vacuum_helper.rb

```diff
@@ -54,6 +54,9 @@ def self._vacuum_custom_index(model, index_name)
 
     model.redis.scan_each_shard("#{custom_index_content_key}#{key_suffix}") do |key|
       hash_tag = key.split(custom_index_content_key)[1] || ""
+      if index_name == nil
+        raise TypeError "index_name nil for model: #{model}"
```

Comment:
Thanks for looking into this.

I think instead of raising a special error, it's better to improve the logging we have here: https://github.com/chanzuckerberg/redcord/blob/43869ee921d5b760d3da19170f4f77dc8d6b7e58/lib/redcord/vacuum_helper.rb#L12

Ideally, it should output both the name of the model and the name of the index to STDOUT.